api-spec/README.md

```diff
@@ -33,8 +33,7 @@ pageable links to iterate through any results past that limit.
 ## Dynamic Catalog API
 
 The other endpoint that is included as an option in STAC API is `/stac/`, which implements the [STAC Catalog Spec](../catalog-spec/README.md).
-See the [Dynamic Catalog](https://github.com/radiantearth/stac-spec/blob/structure_and_cleanup/catalog-spec/catalog-spec.md#dynamic-catalogs)
-section of that spec for more information. Implementing this enables tools like

```

Comment:
Link is broken, not sure if there is need to reference a specific section in catalog-spec/README.md